examples/asr/speech_classification/speech_to_label.py

```diff
@@ -119,27 +144,32 @@
 import pytorch_lightning as pl
 from omegaconf import OmegaConf
 
-from nemo.collections.asr.models import EncDecClassificationModel
+from nemo.collections.asr.models import EncDecClassificationModel, EncDecSpeakerLabelModel
 from nemo.core.config import hydra_runner
 from nemo.utils import logging
 from nemo.utils.exp_manager import exp_manager
 
 
 @hydra_runner(config_path="../conf/matchboxnet", config_name="matchboxnet_3x1x64_v1")
 def main(cfg):
+
     logging.info(f'Hydra config: {OmegaConf.to_yaml(cfg)}')
 
     trainer = pl.Trainer(**cfg.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
-    asr_model = EncDecClassificationModel(cfg=cfg.model, trainer=trainer)
+
+    if cfg.name == 'TitaNet':
```

Comment:
add lower and see if `titanet` in cfg.name.lower(), so they can have their own config with titanet append or prepended to it

Reply:
yeah good point. Updated it.

Reply:
So this is a temp trick. I didn't add task conditions here because we will refactor the model later.